### ISSUE TITLE
l10n Bad translation for "Sort by name" in French 

### ISSUE THREAD

**Issue by StefInP** [nextcloud/ios#2932]
### Steps to reproduce

1. Open the menu in "Files" or "Favorites"

### Expected behaviour

"Sort by > Name" should be translated as "Trier par > **Nom**"

### Actual behaviour

"Sort by > Name" is actually translated as "Trier par > **No**"

### Logs

```
If applicable, you can post the iOS app or server logs (removing any sensitive information).
```

### Reasoning or why should it be changed/implemented?
It seems to be an error in the file "Localizable.strings".
Line 495 : `"_name_"                        = "No";` should be `"_name_"                        = "Nom";`
Sorry I'm new to Github and I don't have the skills to propose the fix.

### Environment data

**iOS version:** iOS 17.4.1

**Nextcloud iOS app version:** 5.3.0

**Server operating system:**

**Web server:**

**Database:**

**PHP version:**

**Nextcloud version:** 28.0.5

**Comment by StefInP** [comment]
I fixed it at Transifex.
Should the issue be closed ?

**Comment by StefInP** [comment]
Fixed with iOS app 5.4.0
